qr-code/src/test/java/com/adyen/checkout/qrcode/DefaultQRCodeDelegateTest.kt

```diff
@@ -265,6 +293,17 @@ internal class DefaultQRCodeDelegateTest(
         }
     }
 
+    @Test
+    fun `download qr image successfully`(): Unit = runBlocking {
```

Comment:
Method name should be a bit more descriptive

Reply:
Minor: `: Unit` is not needed

Reply:
Hmm, i just noticed you're using `runBlocking`, why is that?

Reply:
because `downloadQRImage` is a suspended function, but after the changes I will add with the `eventFlow` it won't be a suspended anymore.

Reply:
Hmm so `runTest` doesn't work with suspend then?